cmd/dkvsrv/main.go

```diff
@@ -345,13 +345,14 @@ func newKVStore() (storage.KVStore, storage.ChangePropagator, storage.ChangeAppl
 
 	dataDir := path.Join(dbFolder, "data")
 	slg.Infof("Using %s as data directory", dataDir)
+
+	sstDir := path.Join(dbFolder, "sst")
```

Comment:
Isn't the data folder already holding the SST files. Rather, why do we need this ?

Reply:
This is required for snapshot creation. Earlier this was passed in L350-354, which is now moved up, so that the value can be reused for both rocksdb & badger.